### PR TITLE
Add Magic Eden Wallet

### PR DESCRIPTION
**Pull request by safaiyeh**
image

### CODE REVIEW

File: packages/wallets/magiceden/src/adapter.ts [hunk 1]
```diff
@@ -0,0 +1,249 @@
...
+    signTransaction(transaction: Transaction): Promise<Transaction>;
+    signAllTransactions(transactions: Transaction[]): Promise<Transaction[]>;
+    signAndSendTransaction(
+        transaction: Transaction,
+        options?: SendOptions
+    ): Promise<{ signature: TransactionSignature }>;
+    signMessage(message: Uint8Array): Promise<{ signature: Uint8Array }>;
+    connect(): Promise<void>;
+    disconnect(): Promise<void>;
+    _handleDisconnect(...args: unknown[]): unknown;
```
Comment (jordansexton) [inline packages/wallets/magiceden/src/adapter.ts:39]: This unused private method looks like it was carried over from the Phantom adapter.

**Review by jordansexton — CHANGES_REQUESTED** [review]
Please keep the changes to the adapter itself, but overall looks great!

**Comment by safaiyeh** [comment]
Thanks for the review @jordansexton! Updated the changes

**Comment by jordansexton** [comment]
Thanks! This will be released momentarily.

**Comment by safaiyeh** [comment]
Hey @jordansexton ! When we looking at to deploy the new wallet adapter?

**Comment by jordansexton** [comment]
Hey, probably tomorrow. As I was getting the project ready for some releases yesterday, I got pulled into a war room about the nascent wallet security issue.

**Comment by jordansexton** [comment]
Update: this will ship tomorrow. https://github.com/solana-labs/wallet-adapter/pull/501 needed to land to fix some issues with the blockhash coming from the connection not being found in preflight.

**Pull request merged**